src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,137 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(last, value, null);
```
![image](https://user-images.githubusercontent.com/20326858/236444618-cda9b74a-7980-4312-9eda-1f3ec971eaad.png)